JhipsterXamarin/JhipsterXamarin/JhipsterXamarin/ViewModels/LoginViewModel.cs

```diff
@@ -61,23 +64,48 @@ public bool RememberMe
             }
         }
 
+        public bool Success
+        {
+            get => _success;
+            set
+            {
+                _success = value;
+                RaisePropertyChanged(() => Success);
+                RaisePropertyChanged(() => Failed);
+            }
+        }
+
+        public bool Failed { get => !_success; }
+
         public LoginViewModel(IMvxNavigationService navigationService, IAuthenticationService authenticationService)
         {
             _navigationService = navigationService;
             _authenticationService = authenticationService;
 
+            Success = true;
+
             SignIn = new MvxCommand(async () =>
             {
                 Active = false;
-                var success = await SignInConnection();
+                Success = await SignInConnection();
                 
-                if (success)
+                if (Success)
                 {
                     await _navigationService.Navigate<HomeViewModel>();
                     await _navigationService.Close(this);
                 }
                 Active = true;
             });
+
+            SignUp = new MvxCommand(async () =>
+            {
+                await _navigationService.Navigate<RegisterViewModel>();
+            });
+
+            ChangeStateCommand = new MvxCommand(() =>
+            {
+                RememberMe = !RememberMe;
+            });
         }
```

Comment:
Use method for each command rather than lambda method

Reply:
Can you give me an example ?

Reply:
`public IMvxCommand LoginCommand => new MvxCommand(LoginClicked);

private void LoginClicked()
{
}
`

Reply:
Replaced in 4f9caa8